product_docs/docs/efm/4/06_monitoring_efm_cluster.mdx

```diff
@@ -55,11 +55,11 @@ Standby database(s) in sync with primary. It is safe to promote.
 The cluster status section provides an overview of the status of the agents that reside on each node of the cluster:
```

Comment:
Specifically, this says the status of the agents, but the table shown doesn't give an agent status, only an agent type. You may be able to infer the agent status from the DB status but....

Reply:
Correct, the text is just referring to the general set of info about the agent: what type it is, does it have the VIP, whatever else. So I think the existing text is still ok.